helm/values.yaml

```diff
@@ -4,7 +4,21 @@ global:
   metrics: {}
   environment: {}
   ingress:
-    domain: 'apps.aroapp.io'  
+    domain: 'apps.aroapp.io'
+  ca:
+    secretName: ''
+    path: '/usr/local/share/ca-certificates'
```

Comment:
shouldn't we keep those values empty so they can be overriden from sgared-deployments?

Reply:
in this case(the ca exist only in global) we don't need to keep the value empty, the umbrella global will override it.